packages/gatsby/src/utils/eslint-config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  emitWarning: true,
+  useEslintrc: false,
+  baseConfig: {
+    globals: {
+      graphql: true,
+    },
+    extends: `react-app`,
+    plugins: [`graphql`],
```

Comment:
You need to connect this too our GraphQL schema still right?

Reply:
The schema is in the store so should be easy to pop into here: https://github.com/gatsbyjs/gatsby/blob/70e80fd5a561aa94acb6fc50f6872aa77714a7ac/packages/gatsby/src/schema/index.js#L27

Reply:
ah good catch! thank you! 🙏